src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,15 @@
+.footer {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  height: 320px;
```

Comment:
try to store such values in variables

Reply:
Thanks for your advices!
Sorry, but I dont fully understand what you mean. What exactly should I store in variables?

Reply:
replied in slack